srcpkgs/taplo/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'taplo'
 pkgname=taplo
-version=0.7.0
+version=0.8.0
 revision=1
-archs="x86_64* i686* aarch64* arm*" # ring
 build_wrksrc="crates/taplo-cli"
 build_style=cargo
+configure_args="--no-default-features --features lsp,native-tls"
```

Comment:
could you leave a comment about why this is necessary?

Reply:
Sure, just added some comments in the latest push.